pkgs/development/python-modules/first.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+let
+  pname = "first";
+  version = "2.0.1";
+in
+buildPythonPackage {
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pn9hl2y0pz61la1xhkdz6vl9i2dg3nh0ksizcf0f9ybh8sxxcrv";
+  };
+
+  meta = with stdenv.lib; {
+    description = "The function you always missed in Python";
+    homepage = https://github.com/hynek/first/;
```

Comment:
Somebody did all this work to publish a function as trivial as `first`? Wow. 
Why would you want to use this instead of `toolz.first` along with `filter? 

Anyway, `doCheck = false;` is missing because the tests aren't included in the archive.